cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Twitter Bootstrap
+or jQuery. Since Symfony isn't purely a back-end framework, it can't deliver
+you much help with Bower. Fortunately, it is very easy to use!
+
+Installing Bower
+----------------
+
+Bower_ is build on top of NodeJS_. Make sure you have that installed and
+then run:
+
+.. code-block:: bash
+
+    $ npm install -g bower
+
+After this command succeeded, run ``bower`` in your terminal to find out if
+it's installed correctly.
+
+.. tip::
+
+    If you don't want to have NodeJS on your computer, you can also BowerPHP_
+    for an unofficial PHP port. Beware that this is still in an alpha state. If
+    you're using BowerPHP, use ``bowerphp`` instead of ``bower`` in the
+    examples.
+
+Configuring Bower in your Project
+---------------------------------
+
+By default, Bower will install all front-end dependencies in a
+``bower_packages/`` in the root of your project. In Symfony, only the ``web/``
+directory has public access, so you have to configure Bower to install it in
+the ``web/`` directory instead.
+
+A common practice is to put your front-end dependencies in
+``web/assets/vendor/``. You can configure this directory in a ``.bowerrc`` file
+in the root of your project. It's using the JSON syntax:
+
+.. code-block:: json
+
+    {
+        "directory": "web/assets/vendor/"
+    }
+
+It is also recommended to create a ``bower.json`` file which will hold your
+projects front-end dependencies (just like ``composer.json`` for Composer). You
+can create an empty file or use ``bower init`` to fill in some information
+about your project.
+
+An Example: Installing Bootstrap
+--------------------------------
+
+Believe it or not, but you're now ready to use Bower in your Symfony
+application. As an example, you'll now install bootstrap in your project and
+include it in your layout.
+
+Installing the Dependency
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first step obviously is to install bootstrap. You can do this by executing
+``bower install``:
+
+.. code-block:: bash
+
+    $ bower install --save bootstrap
+
+This will install Bootstrap and its dependencies in ``web/assets/vendor/`` (or
+another directory you configured).
+
+It'll also save the package name and a version constraint in the ``bower.json``
```

Comment:
maybe we should replace `in the bower.json you created` by `in the bower.json file you created`